### DISCUSSION TITLE
✅ 🔗 Video optimization (Duplicate of #17308)

### DISCUSSION THREAD

**Discussion by ikanc** [elementor/elementor#14132] [Feature Request]
Hey,

I had an issue today where adding a video to a page already puts the video in "standby" mode (preloading) even when I have an image overlay over it.

The issue I'm facing is that the page ranking and size are affected, even though I want the video to only be activated when the user is asking for it.

This could be fixed by having an option to "not preload", and have the video src only be applied to the video **after** the user clicked on the image to start playing.

This has a major impact on the site ranking as I have 30MB video in the homepage which is being counted by Google.

Thanks!

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was introduced with Elementor **v3.10**! 🥳 

You can now select the preload attribute for self-hosted videos. 💪 

✔️ Please feel free to check it out and update your plugin to the new version!

Cheers 🥂
